test/specs/lightbox.js

```diff
@@ -1,11 +1,66 @@
-/* eslint-env jasmine */
-
+import $ from 'jquery';
 import lightbox from '../../lightbox';
 
 describe('lightbox', function() {
-  describe('init', function() {
-    it('should be a function', function() {
-      expect(lightbox.init).toEqual(jasmine.any(Function));
+  const FADE_TIME = 200;
+  const BLOCKING_CLASS = '.js-blocking';
+  const LIGHTBOX_CLASS = '.js-lightbox';
+  const NEXT_CLASS = '.js-next';
+  const CLOSE_CLASS = '.js-close';
```

Comment:
I don't like "running code" outside of the `beforeEach` and `afterEach` personally. Simple assignments are ok, but might lead to copypasta slippery slope to running more complicated code outside of describe, which effects what gets running when you're trying to `fdescribe`, and is ambiguous in terms of overall execution order. :shrugs: